addons/prometheus/template/generate.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+set -euo pipefail
+
+VERSION=""
+function get_latest_release_version() {
+    VERSION=$(helm show chart prometheus-community/kube-prometheus-stack | \
+        grep -i "^appVersion" | \
+        grep -Eo "[0-9]\.[0-9]+\.[0-9]+")
+}
+
+function generate() {
+    # make the base set of files
+    mkdir -p "../${VERSION}"
+    cp -r ./base/* "../${VERSION}"
+    mkdir -p "../${VERSION}/crds"
+    mkdir -p "../${VERSION}/operator"
+
+    # get a copy of the stack with CRDs
+    helm template replaceme prometheus-community/kube-prometheus-stack --values ./values.yaml -n monitoring --include-crds > "../$VERSION/crds/crds-all.yaml"
```

Comment:
i still think you should add version to these

Reply:
The problem is that the version of the chart is something like "14.4.0" while the app version is "0.46.0"

Reply:
but done